docs/integrations-guides/rollup-guides/integrations-overview.md

```diff
@@ -77,70 +87,64 @@ is disabled, and state roots cannot be challenged. This means the sequencer can
 
 An instructive way to dive into the L2 inbox certificate verification strategy
 is to follow an L2 transaction from origination to finalization on Ethereum. We
 can further break this down into two stages, L2 chain finalization and L2 bridge
 finalization.
 
-### L2 Chain Finalization
-
-![M1 chain finalization](/img/eigenda/optimistic-M1-dispersal.png)
+**L2 Chain Finalization**
 
 First, L2 chain finalization. An L2 transaction is finalized with respect to the
-L2 chain when the transaction has been sequenced in the L2 inbox contract. When
-this process is complete, any L2 node can say with confidence that the
-transaction is part of the canonical L2 chain and is not subject to a reorg.
-
-For example, if you were selling your car and a buyer paid you by sending you
+L2 chain when the transaction has been included in the [rollup-inbox][glossary-rollup-inbox] 
+in a finalized L1 block. When this process is complete, any L2 node can say with confidence that the
+transaction is part of the canonical L2 chain and is not subject to a reorg. For example, 
+if you were selling your car and a buyer paid you by sending you
 USDC on a secure rollup, it would be important to wait until the transaction had
 reached L2 chain finalization before letting them drive away with your vehicle.
 
-Let's walk through how EigenDA affects this process:
+![M1 chain finalization](../../../static/img/integrations/secure/inbox-verified-dispersal.png)
 
-1. The batcher component of the rollup sequencer prepares an L2 batch of user
-transactions and sends it to the EigenDA disperser.
-2. The disperser erasure-encodes the blob into chunks, calculates the KZG
-commitment, and calculates the KZG proof for each chunk. It then distributes the
-chunks to the EigenDA operator set, where each operator receives a subset of
-the chunks in proportion to its stake. Each operator then stores the chunks its
-received, verifying that each chunk matches its KZG proof and KZG commitment.
-If so, it signs a message certifying that the chunk has been stored and returns
-it to the disperser.
-3. The disperser aggregates the signatures from step 3 into a single BLS
-signature and sends it and some blob metadata to to the EigenDA Manager contract on
-Ethereum. The EigenDA Manager contract on Ethereum is responsible for verifying EigenDA
-certificates, and if they verify, recording that verification in storage.
-Verification consists of ensuring the aggregated signature is valid and is
-based on the current EigenDA operator set. This blob verification status is
-used in step 5.
-4. Once the blob has been confirmed, the batcher sends a transaction to the
-rollup inbox contract on Ethereum with the EigenDA blob certificate as calldata.
+The above diagram is exactly the same as the trusted integration diagram [above](#trusted-integration), with the added 6th step:
+
+4. In order to get a fully secured integration, the batcher should wait until the confirmBatch tx
```

Comment:
You say above that it's the same, but "with the added 6th step". But there are *two* steps listed here, not one.

Reply:
[fefbad5](https://github.com/Layr-Labs/eigenda-docs/pull/72/commits/fefbad5f827c97c9f4c0ace505e086e4586bd219)